src/main/java/controller/AbstractController.java

```diff
@@ -0,0 +1,24 @@
+package controller;
+
+import http.request.HttpRequest;
+import http.request.method.HttpMethod;
+import http.response.HttpResponse;
+
+import static http.request.method.HttpMethod.GET;
+import static http.request.method.HttpMethod.POST;
+
+public abstract class AbstractController implements Controller {
+
+    @Override
+    public void execute(HttpRequest httpRequest, HttpResponse httpResponse) {
+        HttpMethod httpMethod = httpRequest.getHttpMethod();
+
+        if (httpMethod == GET) {
+            doGet(httpRequest, httpResponse);
+        }
```

Comment:
HTTP method 가 하나 이상일린 없으니 미리 반환해도 될 것 같아요.
```suggestion
        if (httpMethod == GET) {
            doGet(httpRequest, httpResponse);
            return;
        }
```

Reply:
넵 그렇네요ㅎㅎ 수정했습니다!